project.yaml

```diff
@@ -57,5 +57,5 @@ dataSources:
             type: "/cosmwasm.wasm.v1.MsgExecuteContract"
             # Filter to only messages with the vote function call
             contractCall: "swap" # The name of the contract function that was called
-            values: # This is the specific smart contract that we are subscribing to
-              contract: "fetch1qxxlalvsdjd07p07y3rc5fu6ll8k4tmetpha8n"
+#            values: # This is the specific smart contract that we are subscribing to
```

Comment:
Is this address not specific to the instance of an uploaded bridge contract?

Reply:
IIRC, the reliability of the correctness of this address has something to do with it being (deterministically) derived in a way that was constant, at least in the local / testing environments. I don't have the details available at the moment and am not certain whether we have done anything different in this construction to disturb this arrangement.